docs/users/fbc_workflow.md

```diff
@@ -0,0 +1,154 @@
+To generate a final catalog for an operator a user needs to execute different `opm`
+commands based on the template type. We as operator pipeline maintainers want
+to simplify this process and we prepared a `Makefile` with all pre-configured targets.
+
+To get the `Makefile` follow these steps
+
+```bash
+cd <operator-repo>/operator/<operator-name>
+wget https://raw.githubusercontent.com/redhat-openshift-ecosystem/operator-pipelines/main/fbc/Makefile
+```
+
+The right place for the Makefile is in the operator's root directory
+```
+.
+├── 0.0.1
+│   ├── manifests
+│   └── metadata
+├── catalogs.yaml
+├── catalog-templates
+├── ci.yaml
+├── composite-config.yaml
+└── Makefile
+
+```
+
+You can modify the Makefile based on your needs and use it to generate catalogs by running `make catalog`.
+The command uses the `opm` and converts templates into catalogs. The generated catalogs
+can be submitted as a PR in Github and once the PR is processed changes will be released to the
+OCP index.
+
+```bash
+$ tree catalogs
+catalogs
+├── v4.12
+│   └── aqua
+│       └── catalog.yaml
+├── v4.13
+│   └── aqua
+│       └── catalog.yaml
+├── v4.14
+│   └── aqua
+│       └── catalog.yaml
+├── v4.15
+│   └── aqua
+│       └── catalog.yaml
+└── v4.16
+    └── aqua
+        └── catalog.yaml
+
+```
+
+### Adding new bundle to Catalog
+To add a bundle to the catalog you need to first submit the new version of the operator
+using traditional [PR workflow](./contributing-via-pr.md). The operator pipeline builds,
+tests, and releases the bundle into the registry. At this point, the operator is not available
+in the catalog yet. To add the bundle to the catalog you need to update catalog templates
+and add a bundle pullspec given by pull request comment and open a new pull request with catalog
+changes.
+
+#### SemVer
+For example if I want to add `v1.1.0` bundle into `Fast` channel of a specific catalog I'll
+add it as mentioned in the example below:
+```yaml
+---
+Schema: olm.semver
+GenerateMajorChannels: true
+GenerateMinorChannels: true
+Candidate:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v0.1.0
+  - Image: quay.io/foo/olm:testoperator.v0.1.1
+  - Image: quay.io/foo/olm:testoperator.v0.1.2
+  - Image: quay.io/foo/olm:testoperator.v0.1.3
+  - Image: quay.io/foo/olm:testoperator.v0.2.0
+  - Image: quay.io/foo/olm:testoperator.v0.2.1
+  - Image: quay.io/foo/olm:testoperator.v0.2.2
+  - Image: quay.io/foo/olm:testoperator.v0.3.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.1
+  - Image: quay.io/foo/olm:testoperator.v1.1.0
+Fast:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v0.2.1
+  - Image: quay.io/foo/olm:testoperator.v0.2.2
+  - Image: quay.io/foo/olm:testoperator.v0.3.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.0
+  - Image: quay.io/foo/olm:testoperator.v1.1.0 # <-- Add new bundle into fast channel
+Stable:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v1.0.0
+```
+
+#### Basic
```

Comment:
The basic/fbc version of that automate-able step is [here](https://olm.operatorframework.io/docs/advanced-tasks/catalog-update-formulary/#fbc)

Reply:
Added